### PR TITLE
Fix attribution info

### LINKED ISSUE #850: uniquefy copyright information
grafik

### PR DESCRIPTION
**Pull request by taulinger**
fixes #850

### CODE REVIEW

File: test/modules/map/components/attributionInfo/AttributionInfo.test.js [hunk 1]
```diff
@@ -29,360 +29,224 @@ describe('AttributionInfo', () => {
...
-			expect(element.shadowRoot.querySelector('.attribution-container').innerHTML).toContain(arrayAttribution1);
-			expect(element.shadowRoot.querySelector('.attribution-container').innerHTML).not.toContain(arrayAttribution2);
-			expect(element.shadowRoot.querySelector('.is-collapse')).toBeFalsy();
-
-			addLayer(layer2.id, layer2);
-
-			expect(element.shadowRoot.querySelector('.attribution-container').innerHTML).toContain(arrayAttribution2);
-			expect(element.shadowRoot.querySelector('.attribution-container').innerHTML).toContain(arrayAttribution1);
-			expect(element.shadowRoot.querySelector('.is-collapse')).toBeTruthy();
+			// we expect two kinds of attribution: a span conataing a plain string and an anchor element
```
Comment (thiloSchlemmer) [inline test/modules/map/components/attributionInfo/AttributionInfo.test.js:113]: typo con[-a]tai[+n]ing

**Review by thiloSchlemmer — DISMISSED** [review]
THX

File: test/modules/map/components/attributionInfo/AttributionInfo.test.js (hunk 1 above)
Comment (taulinger) [reply test/modules/map/components/attributionInfo/AttributionInfo.test.js:113]: Thx

**Review by thiloSchlemmer — APPROVED** [review]
THX!

**Pull request merged**